Dockerfile-cn

```diff
@@ -0,0 +1,35 @@
+# Only for technical/build aims, built image will be with nginx:alpine according to the last step
+FROM alpine as generate-build-info
+RUN sed -i 's/dl-cdn.alpinelinux.org/mirrors.aliyun.com/g' /etc/apk/repositories \
+    && apk update  \
+    && apk add --no-cache git make
+RUN mkdir -p /usr/src/app/build
+COPY ./Makefile /usr/src/
+WORKDIR /usr/src
+ARG version
+RUN make generate-build-info v=$version
+
+FROM node:14-alpine as build-frontend
+RUN sed -i 's/dl-cdn.alpinelinux.org/mirrors.aliyun.com/g' /etc/apk/repositories \
+    && apk update \
+    && apk add --no-cache python3 make git \
+    && git init
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+COPY ./app/ /usr/src/app/
+RUN export NODE_OPTIONS="--max-old-space-size=4096" \
+    && npm install -g node-gyp --registry=https://registry.npmmirror.com \
+    && export SASS_BINARY_PATH=/usr/src/app/linux_musl-x64-83_binding.node \
+    && npm ci --registry=https://registry.npmmirror.com \
+    # && npx browserslist@latest --update-db \
+    # && npm install --save eslint-plugin-react-hooks@^1.7.0 react-dom@^15.4.2 redux@^3.1.0 redux-first-router@^2.1.1 stylelint@^13.13.0 \
+    && npm run build \
+    && npm run test
+
+FROM nginx:alpine
+COPY --from=build-frontend /usr/src/app/build /usr/share/nginx/html
+COPY --from=generate-build-info /usr/src/app/build /usr/share/nginx/html
+RUN rm /etc/nginx/conf.d/default.conf
+COPY nginx.conf /etc/nginx/nginx.conf
+
+EXPOSE 8080
```

Comment:
Could you explain why our `Dockerfile-full` is not suitable for you?
We use this file for development purposes only. `Dockerfile` is used to run the latest version of the application, which is sufficient in most cases.

Reply:
I got a lot of errors when i use Dockerfile-full, so i changed something(Dockerfile-cn)
I will delete it, and use it locally.

Reply:
Hi, @AmsterGet 

deleted the files, thanks.

and another question: do i need to add zh. to the file(app/.storybook/config.js)?

Reply:
Hello @kisscelia !
Could you please also make a rebase under develop branch to remove unnecessary commits from the PR?

Reply:
hard to me to do this,  :-(

I want to re-fork the repo and make a PR under the latest code of the develop branch...